From de61998c23c0c380ed3ebf874c60de5399a0085c Mon Sep 17 00:00:00 2001
From: Richard Feliciano <RFeliciano@BeaconEmbedded.com>
Date: Tue, 11 May 2021 18:27:51 -0500
Subject: [PATCH 27/27] Change the default Display to be HDMI

The defconfigs for the various imx8M mini and Nano platfors were
changed so the hdmi is now the default display

Signed-off-by: Richard Feliciano <RFeliciano@BeaconEmbedded.com>
---
 configs/beacon_imx8mm_defconfig         | 2 +-
 configs/beacon_imx8mm_fspi_defconfig    | 2 +-
 configs/beacon_imx8mn_2g_defconfig      | 2 +-
 configs/beacon_imx8mn_defconfig         | 2 +-
 configs/beacon_imx8mn_fspi_2g_defconfig | 2 +-
 configs/beacon_imx8mn_fspi_defconfig    | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/configs/beacon_imx8mm_defconfig b/configs/beacon_imx8mm_defconfig
index ca250c1bda..49e7b642a7 100644
--- a/configs/beacon_imx8mm_defconfig
+++ b/configs/beacon_imx8mm_defconfig
@@ -19,7 +19,7 @@ CONFIG_SPL_FIT_GENERATOR="arch/arm/mach-imx/mkimage_fit_atf.sh"
 CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/imx8m/imximage.cfg,SPL_TEXT_BASE=0x7E1000"
 # CONFIG_USE_BOOTCOMMAND is not set
 CONFIG_CONSOLE_MUX=y
-CONFIG_DEFAULT_FDT_FILE="beacon-imx8mm-kit.dtb"
+CONFIG_DEFAULT_FDT_FILE="beacon-imx8mm-hdmi.dtb"
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_BOARD_EARLY_INIT_F=y
diff --git a/configs/beacon_imx8mm_fspi_defconfig b/configs/beacon_imx8mm_fspi_defconfig
index 17f14a6de0..b042efdf3b 100644
--- a/configs/beacon_imx8mm_fspi_defconfig
+++ b/configs/beacon_imx8mm_fspi_defconfig
@@ -19,7 +19,7 @@ CONFIG_SPL_FIT_GENERATOR="arch/arm/mach-imx/mkimage_fit_atf.sh"
 CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/imx8m/imximage_fspi.cfg,SPL_TEXT_BASE=0x7E2000"
 # CONFIG_USE_BOOTCOMMAND is not set
 CONFIG_CONSOLE_MUX=y
-CONFIG_DEFAULT_FDT_FILE="beacon-imx8mm-kit.dtb"
+CONFIG_DEFAULT_FDT_FILE="beacon-imx8mm-hdmi.dtb"
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_BOARD_EARLY_INIT_F=y
diff --git a/configs/beacon_imx8mn_2g_defconfig b/configs/beacon_imx8mn_2g_defconfig
index 18b395e917..82376a9644 100644
--- a/configs/beacon_imx8mn_2g_defconfig
+++ b/configs/beacon_imx8mn_2g_defconfig
@@ -21,7 +21,7 @@ CONFIG_SPL_FIT_GENERATOR="arch/arm/mach-imx/mkimage_fit_atf.sh"
 CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/imx8m/imximage.cfg,SPL_TEXT_BASE=0x912000"
 # CONFIG_USE_BOOTCOMMAND is not set
 CONFIG_CONSOLE_MUX=y
-CONFIG_DEFAULT_FDT_FILE="beacon-imx8mn-kit.dtb"
+CONFIG_DEFAULT_FDT_FILE="beacon-imx8mn-hdmi.dtb"
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_BOARD_EARLY_INIT_F=y
diff --git a/configs/beacon_imx8mn_defconfig b/configs/beacon_imx8mn_defconfig
index 261619d6e0..6de7e5146b 100644
--- a/configs/beacon_imx8mn_defconfig
+++ b/configs/beacon_imx8mn_defconfig
@@ -20,7 +20,7 @@ CONFIG_SPL_FIT_GENERATOR="arch/arm/mach-imx/mkimage_fit_atf.sh"
 CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/imx8m/imximage.cfg,SPL_TEXT_BASE=0x912000"
 # CONFIG_USE_BOOTCOMMAND is not set
 CONFIG_CONSOLE_MUX=y
-CONFIG_DEFAULT_FDT_FILE="beacon-imx8mn-kit.dtb"
+CONFIG_DEFAULT_FDT_FILE="beacon-imx8mn-hdmi.dtb"
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_BOARD_EARLY_INIT_F=y
diff --git a/configs/beacon_imx8mn_fspi_2g_defconfig b/configs/beacon_imx8mn_fspi_2g_defconfig
index a90a982552..3cd82e81c0 100644
--- a/configs/beacon_imx8mn_fspi_2g_defconfig
+++ b/configs/beacon_imx8mn_fspi_2g_defconfig
@@ -21,7 +21,7 @@ CONFIG_SPL_FIT_GENERATOR="arch/arm/mach-imx/mkimage_fit_atf.sh"
 CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/imx8m/imximage_fspi.cfg,SPL_TEXT_BASE=0x912000"
 # CONFIG_USE_BOOTCOMMAND is not set
 CONFIG_CONSOLE_MUX=y
-CONFIG_DEFAULT_FDT_FILE="beacon-imx8mn-kit.dtb"
+CONFIG_DEFAULT_FDT_FILE="beacon-imx8mn-hdmi.dtb"
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_BOARD_EARLY_INIT_F=y
diff --git a/configs/beacon_imx8mn_fspi_defconfig b/configs/beacon_imx8mn_fspi_defconfig
index 54430cefab..e9be4614cf 100644
--- a/configs/beacon_imx8mn_fspi_defconfig
+++ b/configs/beacon_imx8mn_fspi_defconfig
@@ -20,7 +20,7 @@ CONFIG_SPL_FIT_GENERATOR="arch/arm/mach-imx/mkimage_fit_atf.sh"
 CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/imx8m/imximage_fspi.cfg,SPL_TEXT_BASE=0x912000"
 # CONFIG_USE_BOOTCOMMAND is not set
 CONFIG_CONSOLE_MUX=y
-CONFIG_DEFAULT_FDT_FILE="beacon-imx8mn-kit.dtb"
+CONFIG_DEFAULT_FDT_FILE="beacon-imx8mn-hdmi.dtb"
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_BOARD_EARLY_INIT_F=y
-- 
2.17.1

